unittests/AST/ASTImporterGenericRedeclTest.cpp

```diff
@@ -596,21 +665,36 @@ ASTIMPORTER_INSTANTIATE_TYPED_TEST_CASE(RedeclChain, VariableTemplate, ,
 ASTIMPORTER_INSTANTIATE_TYPED_TEST_CASE(RedeclChain, FunctionTemplateSpec, ,
                                         ImportPrototypeThenProtoAndDefinition)
 
-INSTANTIATE_TEST_CASE_P(ParameterizedTests, RedeclChainFunction,
+ASTIMPORTER_ODR_INSTANTIATE_TYPED_TEST_CASE(RedeclChain, Class, Conservative, ,
+                                            DoNotImportConflictingDefinition)
+ASTIMPORTER_ODR_INSTANTIATE_TYPED_TEST_CASE(RedeclChain, Class, Liberal, ,
```

Comment:
Will we have the Liberal test cases instantiated for other kind of Decls too?

Reply:
Yes, more tests follow. But there may be different cases for different types.